cmd/osv-scanner/main.go

```diff
@@ -97,6 +97,9 @@ EXAMPLES:
 	# Scan a source directory
 	$ {{.Name}} scan source -r <source_directory>
 
+	# Scan a source directory in offline mode
+	$ {{.Name}} scan source --offline -r <source_directory>
```

Comment:
I think let's add download offline databases here in the flags as well to let people know how to download the dbs.